yarn/tests/conftest.py

```diff
@@ -32,7 +34,11 @@ def dd_environment():
     with docker_run(
         compose_file=os.path.join(HERE, "compose", "docker-compose.yaml"),
         mount_logs=True,
-        conditions=[CheckEndpoints(INSTANCE_INTEGRATION['resourcemanager_uri'], attempts=240), run_yarn_app],
+        conditions=[
+            CheckEndpoints(INSTANCE_INTEGRATION['resourcemanager_uri'], attempts=240),
+            CheckEndpoints(urljoin(INSTANCE_INTEGRATION['resourcemanager_uri'], YARN_NODES_PATH), attempts=240),
```

Comment:
Would it make sense to just keep this check?

Reply:
yes and I also think we could add some other endpoint, just in case even if the yarn check is not that flaky in general. On it!